.circleci/config.yml

```diff
@@ -23,14 +41,28 @@ jobs:
       - image: python:3.7.4-slim
     steps:
       - run: pip install pyopenssl twine
-      - checkout
```

Comment:
So that is for `aio` code only. Since the workflow lints and packages both `aio` and `rest` code in a single iteration, we need a `nox` with `checkout` for `aio` and a `nox` with an attached `workspace` containing massaged code - `nox-rest`:   https://github.com/talkiq/gcloud-aio/blob/c06802930b12de9485daa0e5649827d6a40bab4e/.circleci/config.yml#L138

I will move `nox-rest` up to `nox` so its side by side for future reference.